keps/sig-node/24-apparmor/README.md

```diff
@@ -0,0 +1,790 @@
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
+
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
+
+  The [Version Skew Strategy](#version-skew-strategy) section covers this point.
+  Running workloads should have no impact as the Kubelet will support either the
+  existing annotations or the new fields introduced by this KEP.
+
+###### What specific metrics should inform a rollback?
+
+  N/A - the feature is already enabled by default since Kubernetes v1.4.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+  Automated tests will cover the scenarios with and without the changes proposed
+  on this KEP. As defined under [Version Skew Strategy](#version-skew-strategy),
+  we are assuming the cluster may have kubelets with older versions (without
+  this KEP' changes), therefore this will be covered as part of the new tests.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+  
+  The promotion of AppArmor to GA would deprecate the beta annotations as described in the 
+  [Version Skew Strategy](#version-skew-strategy).
+
+### Monitoring requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+  
+  The feature is built into the kubelet and api server components. No metric is
+  planned at this moment. The way to determine usage is by checking whether the
+  pods/containers have a AppArmorProfile set.
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ ] Events - Event Reason: 
```

Comment:
This is missing.  This would be handled by pod status set via the kubelet or by events, right?

Reply:
Added details about pod events in [6de8b00](https://github.com/kubernetes/enhancements/pull/3298/commits/6de8b0031e0dfda6aeecab26a0c4e7fc91278de5)